packages/daimo-api/src/api/getAccountHistory.ts

```diff
@@ -149,6 +149,15 @@ function getSuggestedActions(eAcc: EAccount, hist: AccountHistoryResult) {
         url: `daimo://settings/add-passkey`,
       });
     }
+
+    // Active accounts: ask them to join our TG
```

Comment:
looks like this is in the `accountKeys.length === 1` check? want to move it outside?